tasks/rsync-deploy.yml

```diff
@@ -1,19 +1,24 @@
 ---
+
+# Migration check from symlink deployment
+- stat:
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+  register: stat_current_dir
+
+- name: ANSISTRANO | Remove current folder if it's a symlink
+  file:
+    state: absent
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+  when: stat_current_dir.stat.islnk is defined and stat_current_dir.stat.islnk
+
 # Perform rsync deployment
 - name: ANSISTRANO | Ensure current folder is a directory
   file:
     state: directory
     path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
 
 - name: ANSISTRANO | Sync release to new current path
-  synchronize:
-    src: "{{ ansistrano_release_path.stdout }}/"
-    dest: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}/"
-    set_remote_user: "{{ ansistrano_rsync_set_remote_user }}"
-    recursive: yes
-    delete: yes
-    archive: yes
-  delegate_to: "{{ inventory_hostname }}"
+  command: rsync -a -F --delete-after "{{ ansistrano_release_path.stdout }}/" "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}/"
```

Comment:
It starts a ssh connection from the remote server to the remote server (loopback) so the known_hosts file on the remote server will be used.
Maybe i find a way to prevent the usage of the ssh connection but currently it's a big problem :(

The rollback would be just a rsync from a previous release but there is still one issue: the shared folders must be reset if they changed between deployments. If they don't change the `.rsync-filter` will protect the shared folders.
But this functionality still be must be implemented into rollback.